DEVELOPERS.md

```diff
@@ -141,20 +133,20 @@ First, make sure you have the Docker application installed on your device. You c
 3. Run docker
 
    ```sh
-   docker-compose up
+   docker compose up
    ```
 
 This command initializes the containers specified in the docker-compose.yml file. It might take a few moments to complete, depending on your computer and internet connection.
 
-Once the `docker-compose up` process completes, you should have your local version of Supabase up and running within Docker containers. You can access and test the features you've modified or added.
+Once the `docker compose up` process completes, you should have your local version of Supabase up and running within Docker containers. You can access it at `http://localhost:8082`.
```

Comment:
@saltcod - I would maybe reference that you have to have studio running `npm run dev:studio` to access on port 8082